libyara/modules/pe.c

```diff
@@ -1532,11 +1613,14 @@ begin_declarations;
   declare_integer("resource_major_version")
```

Comment:
To be coherent with linker_version, os_version, image_version and subsystem_version, declare an struct "resource_version" containing two fields "major" and "minor".